### PR TITLE
[4.0] Update check alerts

### PR DESCRIPTION
**Pull request by brianteeman**
In J3 the quickicon for the new version etc where hidden in the bottom left of the control panel so we had a popup alert as well. As the quickicons are now bigger and right at the top we don't need the popup as well. Also you will see that the popups over wrote each other so they were even more useless

This PR removes the popup and associated strings and ensures that the icons get the red danger instead of the brown warning and also standardises the 3 strings so that the count is in the same place.

### Testing Instructions
Will require npm i
To test extension updates - install a language and then change the version number in the db
To test joomla updates - create a custom extension server

### Before
updateold

### After

updatenew

### CODE REVIEW

File: build/media_source/plg_quickicon_joomlaupdate/js/jupdatecheck.es6.js [hunk 1]
```diff
@@ -42,24 +42,7 @@
...
-                  ],
-                };
-
-                // Render the message
-                Joomla.renderMessages(messages);
-
-                // Scroll to page top
-                window.scrollTo(0, 0);
-
-                update('warning', Joomla.Text._('PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND').replace('%s', `<span class="badge badge-light"> \u200E ${updateInfo.version}</span>`));
```
Comment (wilsonge) [inline build/media_source/plg_quickicon_joomlaupdate/js/jupdatecheck.es6.js]: This `update` is required

File: build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js [hunk 2]
```diff
@@ -39,24 +39,7 @@
...
-                ],
-              };
-
-              // Render the message
-              Joomla.renderMessages(messages);
-
-              // Scroll to page top
-              window.scrollTo(0, 0);
-
-              update('warning', Joomla.Text._('PLG_QUICKICON_EXTENSIONUPDATE_UPDATEFOUND').replace('%s', `<span class="badge badge-light">${updateInfoList.length}</span>`));
```
Comment (wilsonge) [inline build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js]: same here

File: build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js [hunk 3]
```diff
@@ -39,24 +39,7 @@
...
-  </button>
-</div>`,
-                ],
-              };
-
-              // Render the message
-              Joomla.renderMessages(messages);
-
-              // Scroll to page top
-              window.scrollTo(0, 0);
```
Comment (wilsonge) [inline build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js:57]: If we're removing the scroll to top here (which is fine). Then we can remove the 'requirement' on windows from the top/bottom of the file for the anonymous function as we aren't using it anymore

File: build/media_source/plg_quickicon_joomlaupdate/js/jupdatecheck.es6.js [hunk 4]
```diff
@@ -42,24 +42,7 @@
...
-  </button>
-</div>`,
-                  ],
-                };
-
-                // Render the message
-                Joomla.renderMessages(messages);
-
-                // Scroll to page top
-                window.scrollTo(0, 0);
```
Comment (wilsonge) [inline build/media_source/plg_quickicon_joomlaupdate/js/jupdatecheck.es6.js:60]: same here

File: build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js (hunk 3 above)
Comment (brianteeman) [reply build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js:57]: sorry you will have to explain in dummy terms as I really dont understand js

Comment (brianteeman) [reply build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js:57]: Do you just mean change references of `(window, document, Joomla);` to `(document, Joomla);`

Comment (wilsonge) [reply build/media_source/plg_quickicon_extensionupdate/js/extensionupdatecheck.es6.js:57]: Yes top and bottom of the file

**Comment by brianteeman** [comment]
@wilsonge I updated jupdatecheck.es6.js as suggested

I didnt see anything in extensionupdatecheck.es6.js to update ??

**Comment by wilsonge** [comment]
Looks good to me. Yeah I'm not sure exactly why that is - i guess something is theoretically wrong somewhere there :/ but as long as it works :/

**Comment by brianteeman** [comment]
Thanks

**Comment by infograf768** [comment]
Note:
Taking off the Messages is not the solution imho. We have an underlying issue.
The same situation happens when one updates many items at the same time: the message only displays for the last item updated.
Here for languages:
Install older packs
[de-DE_joomla_lang_full_3.9.5v1.zip](https://github.com/joomla/joomla-cms/files/3181075/de-DE_joomla_lang_full_3.9.5v1.zip)
[fr-FR_joomla_lang_full_3.9.5v2.zip](https://github.com/joomla/joomla-cms/files/3181076/fr-FR_joomla_lang_full_3.9.5v2.zip)

This is what you get when you update:
updates_messages

**Comment by infograf768** [comment]
What I meant is that the ideal for these checks would be to combine the Messages in a unique Warning message with as many items concerned as necessary.

**Comment by brianteeman** [comment]
That was a side effect bonus of the issue that was being solved here. Please create your own issue for the messages

**Comment by wilsonge** [comment]
@infograf768 there's definitely a bigger problem to solve in speed of messages being removed. However in this specific case we had both alerts and the warning buttons at the top of the screen. Which meant we had two alerts for the same thing.

**Pull request merged**